packages/block-library/src/missing/edit.native.js

```diff
@@ -83,11 +97,30 @@ export class UnsupportedBlockEdit extends Component {
 				  __( "'%s' isn't yet supported on WordPress for iOS" );
 		const infoTitle = sprintf( titleFormat, title );
 
+		const actionButtonStyle = getStylesFromColorScheme(
+			styles.actionButton,
+			styles.actionButtonDark
+		);
+
 		return (
 			<BottomSheet
 				isVisible={ this.state.showHelp }
 				hideHeader
-				onClose={ this.toggleSheet.bind( this ) }
+				onClose={ this.toggleSheet }
+				onModalHide={ () => {
+					if ( this.state.sendFallbackMessage ) {
```

Comment:
I'm sure you're already aware of this, but I wish there was a way pass information to `onModalHide` about which action caused the modal to close, without resorting to a boolean `sendFallbackMessage` property on the component. 

When dismissing a modal on iOS we have an animation completion handler, but I don't see an equivalent pattern available with `react-native-modal`.

Reply:
Looking into this with @etoledom we didn't find a better solution for now.